ADotNet/Services/Foundations/Files/FileService.Exceptions.cs

```diff
@@ -22,40 +23,74 @@ private static void TryCatch(ReturningNothingFunction returningNothingFunction)
             {
-                var invalidFileException =
-                    new InvalidFileException(argumentNullException);
-
-                throw new FileDependencyValidationException(invalidFileException);
+                throw CreateFileDependencyValidationException(argumentNullException);
             }
             catch (ArgumentException argumentException)
             {
-                var invalidFileException =
-                    new InvalidFileException(argumentException);
-
-                throw new FileDependencyValidationException(invalidFileException);
+                throw CreateFileDependencyValidationException(argumentException);
             }
             catch (SerializationException serializationException)
             {
-                var failedFileSerializationException =
-                    new FailedFileSerializationException(serializationException);
-
-                throw new FileDependencyException(failedFileSerializationException);
+                throw CreateFileDependencyException(serializationException);
             }
             catch (Exception exception)
             {
-                var failedFileServiceException =
-                    new FailedFileServiceException(exception);
-
-                throw new FileServiceException(failedFileServiceException);
+                throw CreateFileServiceException(exception);
             }
         }
+
+        private static FileValidationException CreateFileValidationException(
+            Xeption innerException)
+        {
+            return new FileValidationException(
+                "File validation error occurred, fix the errors and try again.",
+                innerException);
+        }
+
+        private static FileDependencyValidationException CreateFileDependencyValidationException(
+            Exception innerException)
+        {
+            var invalidFileException =
+                new InvalidFileException(
+                    "Invalid file error occurred.", innerException);
+
+            throw new FileDependencyValidationException(
+                "File dependency validation error occurred, fix the errors and try again.",
+                invalidFileException);
+        }
+        
+        private static FileDependencyException CreateFileDependencyException(
+            Exception innerException)
+        {
+            var failedFileSerializationException =
+                new FailedFileSerializationException(
+                    "Failed file serialization error occurred, contact support.",
+                    innerException);
+
+            throw new FileDependencyException(
+                "File dependency error occurred, contact support.",
+                failedFileSerializationException);
+        }
+        
+        private static FileServiceException CreateFileServiceException(
+            Exception innerException)
+        {
+            var failedFileServiceException =
```

Comment:
We may need to initialize this guy _inside_ the `catch` block becuase a Service Exception can be used for multiple other inner exception types. Same thing goes with other categorical exceptions